tasks/modules.py

```diff
@@ -144,7 +144,7 @@ def dependency_path(self, agent_version):
         "test/new-e2e",
         independent=True,
         should_tag=False,
-        targets=["./runner", "./utils/e2e/client"],
+        targets=["./pkg/runner", "./pkg/utils/e2e/client"],
```

Comment:
Should the second target be limited to `./pkg/utils/e2e/client`? I see a `./pkg/utils/e2e/e2e_test.go` file that wouldn't be covered.

Reply:
The reason why that test is excluded is that it requires pulumi, so it cannot run with the standard go test runner. I'm thinking adding an env var to skip it, so that we can target the whole `new_e2e` and skip anything involving `pulumi`.
What do you think ?